flocker/node/agents/functional/test_cinder_behaviour.py

```diff
@@ -0,0 +1,84 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
```

Comment:
"Test actual real world behaviour of Cinder implementations, so we know when to update our workarounds"? or something